docs/new_flux_ideas_and_guide.md

```diff
@@ -0,0 +1,985 @@
+            again at 0. These wraps and resets need to be accounted for when performing operations on counters. 
+            An example of a counter is the number of bytes a network interface has received. 
+* gauge - A value that measures a specific value, which can increase or decrease. An example of a gauge is 
+          taking a temperature reading.
+
+### Events
+Events are single data points in time. This could be an individual request to an API and how long that took, a user 
+logging into a service, a car parking, a light turning on, a container starting or stopping, or anything you can think 
+of. Events are what are called irregular time series given that they can occur at any time. It is possible to create 
+regular time series from an underlying event stream (like counting how many occurred every five minutes for the last hour).
+
+### Working with data in Flux
+When working with data in Flux, it translates the line protocol/time series structure into to a table. You 
+can think of a measurement name as a table with the tag and field keys representing columns. Flux also 
+includes the reserved column, `time`, in every table pulled from InfluxDB. The `time` column is the time at the end 
+of the line protocol when data is written in.
+
+Here's an example of line protocol and its translation to a Flux table:
+
+```
+h2o_temperature,location=santa_monica,state=CA surface_degrees=65.2,bottom_degrees=50.4 1568756160
+h2o_temperature,location=santa_monica,state=CA surface_degrees=63.6,bottom_degrees=49.2 1600756160
+h2o_temperature,location=coyote_creek,state=CA surface_degrees=55.1,bottom_degrees=51.3 1568756160
+h2o_temperature,location=coyote_creek,state=CA surface_degrees=50.2,bottom_degrees=50.9 1600756160
+h2o_temperature,location=puget_sound,state=WA surface_degrees=55.8,bottom_degrees=40.2 1568756160
+h2o_temperature,location=puget_sound,state=WA surface_degrees=54.7,bottom_degrees=40.1 1600756160
+```
+
+| location | state | surface_degrees | bottom_degrees | time |
+| --- | --- | --- | --- | --- |
+| coyote_creek | CA | 55.1 | 51.3 | 1568756160 |
+| coyote_creek | CA | 50.2 | 50.9 | 1600756160 |
+| puget_sound | WA | 55.8 | 40.2  | 1568756160 |
+| puget_sound | WA | 54.7 | 40.1  | 1600756160 |
+| santa_monica | CA | 65.2 | 50.4 | 1568756160 |
+| santa_monica | CA | 63.6 | 49.2 | 1600756160 |
+
+Note that the measurement name doesn't appear anywhere. That's because the table itself represents data from the 
+`h2o_temperature` measurement. Later we'll see how to work with multiple measurements (i.e. tables). Note that the 
+underlying time series would be represented by the combination of `location`, `state`, and the specific field. 
+Results are ordered by default by the columns of the tags and their values and then by time. So we can see that 
+`coyote_creek`, `CA` rows are together in time ascending order.
+
+We'll show later how to specify to the graphing engine what represents an individual time series that you want 
```

Comment:
what is the graphing engine?

Reply:
I was skeptical of this graphing stuff getting mixed in here. It would be nice to have this direct and focused on making flux easy to use. Would be nice if this focused on  I think adding graphing concerns here takes focus away from that. Just my $0.02 🤷‍♂